dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -865,6 +883,166 @@ public DagDataSchedule exportProcessDagData(ProcessDefinition processDefinition)
         return result;
     }
 
+    @Override
+    @Transactional(rollbackFor = RuntimeException.class)
+    public Map<String, Object> importSqlProcessDefinition(User loginUser, long projectCode, MultipartFile file) {
```

Comment:
It's a big method, can you split into some independent methods?

Reply:
OK, but I'm afraid it's difficult to reuse the method

Reply:
sorry I forgot to submit my review